### PR TITLE
CLI: Add documentation for all commands

### PR DESCRIPTION
**Pull request by Glitch18**
Similar to #86, 
This PR adds help strings and command descriptions to all the CLI commands for warp.

### CODE REVIEW

**Comment by JorikSchellekens** [comment]
@Glitch18 any progress on this?

**Comment by Glitch18** [comment]
Hey Jorik, give me a few more hours. Will update this.

File: warp/cli/warp_cli.py [hunk 1]
```diff
@@ -27,6 +27,12 @@ def warp():
 @click.argument("file_path", type=click.Path(exists=True))
 @click.argument("contract_name")
 def transpile(file_path, contract_name):
+    """
+    FILE_PATH: Path to your solidity contract\n
+    CONTRACT_NAME: Name of the primary contract (non-interface, non-library, non-abstract contract) that you wish to transpile
+
+    Generates the transpiled Cairo contract.
```
Comment (murcake) [inline warp/cli/warp_cli.py:34]: Well, now it generates a json with all the info needed to use that contract, including the transpiled Cairo contract.

File: warp/cli/warp_cli.py [hunk 2]
```diff
@@ -70,10 +83,11 @@ def invoke(program, address, function, inputs):
     type=click.Path(exists=True, dir_okay=False),
 )
 def deploy(program):
-    """Deploy PROGRAM.
-
-    PROGRAM is the path to the transpiled program JSON file.
+    """
+    PROGRAM: path to the transpiled program JSON file.
 
+    Compiles the given Cairo contract and deploys it to StarkNet.
```
Comment (murcake) [inline warp/cli/warp_cli.py:89]: Also not exactly a Cairo contract, but a Cairo program info file.

File: warp/cli/warp_cli.py [hunk 3]
```diff
@@ -92,6 +106,10 @@ def deploy(program):
 @warp.command()
 @click.argument("tx_id", nargs=1, required=True)
 def status(tx_id):
+    """
+    TX_ID: The transaction ID printed to stdout after invoking/deployment.
```
Comment (murcake) [inline warp/cli/warp_cli.py:110]: This should probably be renamed to "tx_hash", because these are hashes now on StarkNet and are called such.

**Review by murcake — APPROVED** [review]

**Comment by murcake** [comment]
Let's rebase once again and focus on merging this PR as the first thing

**Comment by Glitch18** [comment]
> Let's rebase once again and focus on merging this PR as the first thing

On it

**Review by murcake — APPROVED** [review]

File: warp/cli/warp_cli.py [hunk 4]
```diff
@@ -64,10 +77,11 @@ def invoke(program, address, function, inputs):
...
 @click.option("--constructor_args", required=False, default="[]")
 def deploy(program, constructor_args):
-    """Deploy PROGRAM.
-
-    PROGRAM is the path to the transpiled program JSON file.
+    """
+    PROGRAM: path to the transpiled program JSON file.
 
+    Compiles the given Cairo program info file and deploys it to StarkNet.
+    Contract's address will be printed to stdout and saved in a file called CONTRACT_NAME_ADDRESS.txt
```
Comment (murcake) [inline warp/cli/warp_cli.py:84]: I think it's no longer the case that address is saved to a file. It's just printed to stdout

**Review by murcake — APPROVED** [review]

**Pull request merged**